internal/command/action_test.go

```diff
@@ -30,13 +30,13 @@ func TestConvertSubsToMap(t *testing.T) {
 
 		output := convertSubsToMap(input)
 
-		So(output, ShouldResemble, expected)
+		assert.EqualValues(t, expected, output)
```

Comment:
Does `assert.EqualValues()` work for maps? (Reference: https://pkg.go.dev/github.com/stretchr/testify/assert#EqualValues)

_"EqualValues asserts that two objects are equal or convertable to the same types and equal."_

I take the meaning of `Values` here to mean the underlying value of an object after it has been converted to some type, and not the values in a map???

Wouldn't we need something like this?:
```
    assert.True(t, reflect.DeepEquals(expected, output))
```

Reply:
See my comment on the other PR here: https://github.com/cyberark/conjur-api-go/pull/118#discussion_r766900074

Reply:
Wow, great to know!!!